app/components/post-search-results-item.js

```diff
@@ -6,6 +6,20 @@ import Thumbnailer from '../modules/thumbnailer';
 
 export default Component.extend({
   classNames: ['post-search-results-item'],
+
+  // some old posts do not have title, we'll fake it here
+  getTitle: computed('post', function () {
+    if (this.post.title) {
+      return this.post.title;
+    }
+
+    if (this.post.content.length > 100) {
```

Comment:
Any chance this can be null?

Reply:
hmmm...

Reply:
Are you saying no post title and no post content?